library/packages/src/modules/PackageCallbacks.rb

```diff
@@ -758,21 +758,18 @@ def MediaChange(error_code, error, url, product, current, current_label, wanted,
         current_device
       )
 
-      url_tokens = URL.Parse(url)
-      url_scheme = Ops.get_string(url_tokens, "scheme", "")
-      url_scheme = Builtins.tolower(url_scheme)
+      url_scheme = Ops.get_string(URL.Parse(url), "scheme", "").downcase
 
       # true if it makes sense to offer an eject button (for cd/dvd only ...)
-      offer_eject_button = url_scheme == "cd" || url_scheme == "dvd"
+      is_disc = ["cd", "dvd"].include?(url_scheme)
```

Comment:
Funny part is that below everything that is not this is called medium and this is called disc